stream_alert_cli/athena/handler.py

```diff
@@ -249,6 +256,7 @@ def create_table(athena_client, options, config):
             'CREATE EXTERNAL TABLE {table_name} ({schema}) '
             'PARTITIONED BY (dt string) '
             'ROW FORMAT SERDE \'org.openx.data.jsonserde.JsonSerDe\' '
+            'WITH SERDEPROPERTIES ( \'ignore.malformed.json\' = \'true\') '
```

Comment:
Minor nit: additional space before `\'ignore.malformed.json\' = \'true\'`, but no space after it.